enterprise/utils.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to be used across the enterprise djangoapp
+"""
+try:
+    from openedx.core.djangoapps.api_admin.utils import course_discovery_api_client
```

Comment:
What benefit we would get by catching this exception? Shouldn't we just let this import error bubble up so that we know early that we depend on the catalog service for EC's?
Also if we want to disable the EC app then there should be a proper way of doing this like some feature flag or switch.